config/kinto.ini

```diff
@@ -184,6 +184,10 @@ kinto.amo.plugins = /buckets/blocklists/collections/plugins
 kinto.amo.gfx = /buckets/blocklists/collections/gfx
 kinto.amo.certificates = /buckets/blocklists/collections/certificates
 
+kinto.amo.preview.addons = /buckets/blocklists-preview/collections/addons
+kinto.amo.preview.plugins = /buckets/blocklists-preview/collections/plugins
+kinto.amo.preview.gfx = /buckets/blocklists-preview/collections/gfx
+kinto.amo.preview.certificates = /buckets/blocklists-preview/collections/certificates
```

Comment:
Then you can add these to `kinto.signer.resources` ?

Maybe we could perform a `GET` in the `smoke-test.sh` script ?